### ISSUE TITLE
False-positive MULTIPLE_INIT_BLOCKS when properties are assigned in init block using local variables

### ISSUE THREAD

**Issue by petertrr** [cqfn/diKTat#795]
## Describe the bug
```kotlin
class Credentials(auth: String) {
    val gitHubUserName: String
    val gitHubAuthToken: String

    init {
        val splitAuth = auth.split(":")
        if (splitAuth.size != 2) {
            """Incorrect value is passed with 'auth' property.
                    | It should contain user and auth-token separated by a colon (':').
                    | This option is needed to extend the limit of requests.""".trimMargin() logAndExit 8
        }
        gitHubUserName = splitAuth[0]
        gitHubAuthToken = splitAuth[1]
    }
}
```

Now diktat suggests to move property assignments outside of the init block, but a local variable is used for this assignments, so this refactoring is too complicated. Diktat should not emit a warning in this case.

## Environment information
* diktat version: 0.4.2

**Comment by petertrr** [comment]
Should be fixed by #967
